README.rst

```diff
@@ -1,148 +1,262 @@
+================================
 FIWARE Policy Manager GE - Facts
-____________________
+================================
 
 | |Build Status| |Coverage Status| |Pypi Version| |Pypi License|
 
-Description
-===========
 
-This module is part of FIWARE Policy Manager.
+This is the code repository for **FIWARE Policy Manager GE - Facts**, a server to process the incoming facts from the
```

Comment:
Before this content, you should add a index to the different sections, after it you have to add a Section called *Introduction* and start it with:
     "This is the code repository for **FIWARE Policy Manager GE - Facts**, a..."